### DISCUSSION TITLE
How to add audio poster?

### DISCUSSION THREAD

**Discussion by Mister-Hope** [vidstack/player#858] [Q&A]
Does the audio community skin support poster?

The readme image seems having one, but I cannot get the poster displayed when providing an audio source (working with video through)

![image](https://github.com/vidstack/player/assets/33315834/c51c8600-1b91-46d8-9ad6-648e02f648f5)

Also I noticed there are difference between the actual look in my project and readme image.

**Comment by mihar-22** [comment]
We couldn't get the poster to work with the audio player responsively well. The problem with that design is that it relies on certain information like titles and other things being present. When they're not given by the user, the design isn't balanced. We had to revise the audio design so it works in a general way.

You can easily build that audio player yourself using our components. In the future, we'll add example designs which you can copy paste that will include it.

**Reply by Mister-Hope** [reply]
Thanks for the answer, if I am understanding everything well, the readme image is based on the discussion design, while the community skin production drop some of the items?

**Reply by mihar-22** [reply]
Ye, but we can still provide that design as a copy/paste template in the future when we put together more examples. It's really easy to build.
